hw/dv/tools/dvsim/xcelium.hjson

```diff
@@ -199,8 +199,8 @@
     {
       name: xcelium_gui
       is_sim_mode: 1
-      build_opts: ["-createdebugdb", "-access +c"]
-      run_opts: ["-gui"]
+      build_opts: ["-createdebugdb", "-access +c", "-uvmlinedebug", "-linedebug"]
+      run_opts: ["-gui", "-input '@database -open waves -into waves.shm -default;probe -create tb -depth all -packed 32k -unpacked 32k -all -dynamic -database waves'"]
```

Comment:
This change is definitely not something that's needed. You undo it again in a later commit in the PR. Please squash the two for this file. (Easiest approach is probably to do an interactive rebase and pause after this commit. Then `git checkout my-work-branch hw/dv/tool/dvsim/xcelium.hjson` and `git commit --amend` and `git rebase --continue`).